packages/aws-cdk-lib/aws-rds/lib/instance.ts

```diff
@@ -1355,7 +1355,7 @@ export class DatabaseInstanceReadReplica extends DatabaseInstanceNew implements
     }
 
     // The read replica instance always uses the same engine as the source instance
-    // but some CF validations require the engine to be explicitely passed when some
+    // but some CF validations require the engine to be explicitly passed when some
```

Comment:
Similar typo in [security-group](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-ec2/lib/security-group.ts) up for grab